soaks/tests/datadog_agent_remap_datadog_logs/terraform/http_blackhole.toml

```diff
@@ -1,3 +1,3 @@
-worker_threads = 4
+worker_threads = 1
```

Comment:
Just curious if there is some context for this change.

Reply:
Previously the blackhole would use up to 4 threads / CPUs. We want it to use only 1. The original setting was copypasta from my personal machine.

Reply:
Ah, gotcha 👍